orc8r/cloud/go/obsidian/swagger/spec_servicer_test.go

```diff
@@ -34,13 +34,14 @@ var (
 )
 
 func TestSpecServicer_NewSpecServicerFromFile(t *testing.T) {
+	os.RemoveAll(tmpDir)
+	defer os.RemoveAll(tmpDir)
+
 	req := &swagger_protos.GetSpecRequest{}
```

Comment:
One more thing -- can you move this declaration down past all the directory handling, to just before you start making requests of the servicer